.gitlab/deploy_containers/conditions.yml

```diff
@@ -28,6 +28,30 @@
       DSD_REPOSITORY: dogstatsd
       IMG_REGISTRIES: public
 
+.manual_on_deploy_auto_on_rc-ot:
```

Comment:
I actually tried to use the following convention:

`job_name-variant`

So that anything with `_` underscores can be regarded as the actual job name, of which there may be other variants. And the `-` hyphen is used to separate the job name from the particular variant. If you feel strongly about it, I can change it, but I still find it visually useful.

Reply:
I do understand your point. On my side, I often never really know when to put underscore or hyphen, because:
* the convention is not used everywhere (See for instance convention [here](https://github.com/DataDog/datadog-agent/blob/main/.gitlab/packaging/deb.yml#L42), [here](https://github.com/DataDog/datadog-agent/blob/main/.gitlab/e2e_install_packages/amazonlinux.yml#L25) and [here](https://github.com/DataDog/datadog-agent/blob/main/tasks/github_tasks.py#L222)(
* it's not super easy to know what is a variant.

However, this is probably not related to this PR: I was trying to limit some naming conventions, and as there is no strict rule now, I can let you do as you prefer 😄